scripts/repo_metrics/track_metrics.py

```diff
@@ -50,9 +52,27 @@ def parse_args():
         action="store_true",
         help="Whether or not to save the information to the database",
     )
+    parser.add_argument(
+        "--event_date",
+        type=_valid_date,
+        default="now",
+        help="Date for an event (format: YYYY-MM-DD or now)",
+    )
```

Comment:
another option to reduce code would be to leverage dateutil library and remove the _valid_date function
```
from dateutil.parser import isoparse
from datetime import datetime
...
parser.add_argument('--event_date', default=datetime.now().isoformat(), type=isoparse)
```

Reply:
this is so nice! great catch!